draft-ietf-rats-architecture.md

```diff
@@ -360,6 +363,42 @@ are signed.  For example, values might have been generated at boot, and then
 used in claims as long as the signer can guarantee that they cannot have changed
 since boot.
 
+The term freshness is applied to both Evidence and Attestation Results in
+this document. Unfortunately, the term freshness is also a bit overloaded and
+in order to illustrate the application of the term in this document better,
+two characteristics that can be part of freshness are elaborated on below:
+recentness and uniqueness:
+
+Recentness:
+
+: a quality of Evidence that is based on the time that has passed between
+creation of the evidence by an Attester (typically by "sealing" a Claims set
+via signing) and the reception of that Evidence by a Verifier or (relaying
```

Comment:
"sealing via signing" seems over-specific to me (I commented similarly on #85).  I'd drop it and just leave "(typically by sealing a Claims set)"

Reply:
I understand the specifics of a "Secure Channel" that would not require sealing. As this is covered by Giri's PR, I deliberately highlighted "the other"/"most common" way to inhibit tampering here. And I think that is okay, still, taking your comment into account.